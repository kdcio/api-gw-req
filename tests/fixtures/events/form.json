{
  "httpMethod": "POST",
  "body": "message=hello&name=world",
  "requestContext": { "authorizer": "dev" }
}
