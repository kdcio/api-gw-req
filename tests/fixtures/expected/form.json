{
  "path": "",
  "method": "POST",
  "query": {},
  "headers": {},
  "ip": "",
  "userAgent": "",
  "pathParams": "",
  "body": { "message": "hello", "name": "world" },
  "authorizer": "dev"
}
